### DISCUSSION TITLE
Loses settings after power outage

### DISCUSSION THREAD

**Discussion by zeha** [gcormier/megadesk#168] [Q&A] [answered]
I have two Megadesks which I bought mid 2023, so I think they should have the current firmware. For a long time I kept the desks always powered on, so I never noticed any problems.

Now that sometimes power goes out, I've noticed the stored positions are lost after power comes back. It also plays what I think is the sad tone sequence (= eeprom invalid?).

Is there some way of forcing a save of the settings?

Moving works fine.

**Comment by gcormier** [comment]
That's odd indeed. You could try wiping EEPROM (hold DOWN while powering up megadesk) to start. Next step, reupload the firmware to the device.

**Comment by Yveaux** [comment, marked as answer]
#169 may fix your issues too.

**Reply by zeha** [reply]
This is really interesting! Thank you. I'll give it a try soon.

**Reply by zeha** [reply]
I've changed the fuses on the first one and it seems to keep its settings now. Will test a bit more and report back.

**Reply by zeha** [reply]
Can confirm #169 fixes the problem for me, on both desks that consistently lost their settings.
Thank you!
